daffodil-runtime1/src/main/scala/org/apache/daffodil/runtime1/layers/api/Layer.java

```diff
@@ -63,11 +63,11 @@
  * <p/>
  * A layer that wants to return a value after the layer algorithm completes defines a special recognizable
  * getter method. The name of the getter is formed from prefixing the DFDL variable name with the string
- * 'getDFDLResultVariable_'. The return type of the getter must match the type of the variable.
+ * 'getLayerVariableResult_'. The return type of the getter must match the type of the variable.
  * <p/>
  * For example, a result value getter for a DFDL variable named 'checksum' of type xs:unsignedShort would be:
  * <pre>
- *      Int getDFDLResultVariable_checksum() {
+ *      Int getLayerVariableResult_checksum() {
```

Comment:
Do these set/getVariable functions need o be public, or can reflection still find them if they are package private? 

Also, it might be worth using `int` instead of `Int` since this is javadoc.

Reply:
package private works also. I've changed all ours, since those will be used as examples.